### ISSUE TITLE
API response from JSON field always null

### ISSUE THREAD

**Issue by pleodigital** [voorhoede/datocms-plugin-word-counter#12]
Hello!
I'm trying to include Word Counter plugin into my project and I have problem with settings.  I have just two fields: 
![image](https://github.com/voorhoede/datocms-plugin-word-counter/assets/48985121/f70d0610-4dd9-4898-842c-11e2ccd9961c)
 In presentation Tab of my text field I'm adding add-on "Word Counter" and setup like below: 
![image](https://github.com/voorhoede/datocms-plugin-word-counter/assets/48985121/10b8fd58-fd90-49b0-b202-de138e00d7a5)
 (I leave "Exposed word counter field id" empty, but i also tried with typing "text_word_counter" and "some_example_counter"). 
In presentation settings of my JSON field I don't have any option like "word counter" - should I have it? 
![image](https://github.com/voorhoede/datocms-plugin-word-counter/assets/48985121/1f548d22-0934-4ad7-a329-85579675b695)

To sum up, I've tried multiple configurations (also auto apply to fields ON and OFF, install previous version (1.5.2)) but no matter how I configure your plugin, I get "null" from API. Do you have any idea how to solve this problem? Thanks :)

**Comment by velomovies** [comment]
Hi @pleodigital, sorry for the late response!
Thanks for using the plugin and thanks for the clear bug report. I could recreate the issue and saw some weird behaviour when using blocks and the Word counter plugin. 

We fixed this and it should be working correctly now. 

When Dato has picked up the fix it will be available on `v1.6.0`. I hope this fixed your issue. If not feel free to open a new issue!

**Comment by sebashine-branly** [comment]
Hi @velomovies
I think it could also be due to https://github.com/voorhoede/datocms-plugin-word-counter/issues/5#issuecomment-1165340665
